core/templates/dev/head/services/exploration-features.service.ts

```diff
@@ -16,7 +16,18 @@
  * @fileoverview Service for determining the visibility of advanced features in
  *               the exploration editor.
  */
-
+export interface ExplorationDataInterface {
```

Comment:
Add a newline above this. Also for Interfaces the name should start with a capital I so this should be named as ```IExplorationData```